git-hooks/util/formatting.js

```diff
@@ -0,0 +1,80 @@
+// See http://bit.ly/2ORJ0x2 (\x1b sequence)
```

Comment:
I guess this is for later?

Reply:
nah, this module is actually being used in `git-hooks/commit/errors.js` (lines 2-6)